src/t_cose_signature_verify_eddsa.c

```diff
@@ -176,7 +179,11 @@ t_cose_signature_verify_eddsa_init(struct t_cose_signature_verify_eddsa *me,
 {
     memset(me, 0, sizeof(*me));
     me->s.rs.ident   = RS_IDENT(TYPE_RS_VERIFIER, 'E');
+#ifndef T_COSE_DISABLE_SIGN
     me->s.verify_cb   = t_cose_signature_verify_eddsa_cb;
+#else
+    me->s.verify_cb   = NULL;
```

Comment:
The callbacks are not supposed to be NULL per comment in definition of struct t_cose_signature_verify

Reply:
Updated.